app/http/http.yaml

```diff
@@ -0,0 +1,62 @@
+components:
+  schemas:
+    App.Http:
+      description: >-
+        HTTP configs.
+      type: object
+      properties:
+        transport_name:
+          description: >-
+            TCP interface name
+          type: string
+          x-constraint:
+          - '/components/schemas/App.Tcp/properties/name'
+          x-field-uid: 1
+        http_version:
+          description: >-
+            HTTP Version
+          type: integer
+          default: 1
+          x-field-uid: 3
```

Comment:
Why is this '3' and not '2'?  It shouldn't matter from a correctness standpoint.  Just curious as to why it was skipped?